documentation/asciidoc/computers/ai/getting-started.adoc

```diff
@@ -170,6 +170,19 @@ $ sudo apt-mark unhold hailo-tappas-core hailort hailo-dkms
 
 [tabs]
 ======
+v4.19::
+To install version 4.19 of Hailo's neural network tooling, run the following commands:
++
+[source,console]
+---
+sudo apt install hailo-tappas-core=3.30.0-1 hailort=4.19.0-3 hailo-dkms=4.19.0-1 python3-hailort=4.19.0-2
```

Comment:
This now includes `python3-hailort` which wasn't listed as a requirement in earlier versions - is that correct?

Reply:
That's correct.